src/odemis/cli/main.py

```diff
@@ -331,6 +332,32 @@ def print_vattribute(name, va, pretty):
                 logging.info("Failed to convert %s to component names")
                 # Leave the value as-is
 
+        # Convert to nicer unit for user
+        if va.unit and va.unit == "rad" and isinstance(val, numbers.Real):
+            try:
+                val_converted = u" = %s°" % (math.degrees(val),)
+            except Exception:
+                logging.warning("Failed to convert %s to degrees", name)
+                val_converted = u""
+        else:
+            val_converted = u""
+
+        # For position, it's trickier, as the unit is on .axes
+        if (name == "position" and isinstance(va.value, dict) and
+            hasattr(component, "axes") and isinstance(component.axes, dict)
+           ):
+            pos_deg = {}
+            for an, pos in va.value.items():
+                try:
+                    axis_def = component.axes[an]
+                except KeyError:
+                    continue
```

Comment:
It's better to add a statement showing what axis missing: `logging.error("The axis %s is missing" %an)`. 
Such messages are not for the user, but for the developer if wants to change the code in the future. This was proven to me after adapting enzel code for meteor.

Reply:
Adding log statements affect the user, so if it's not for the user, I'd suggest to pass the information a different way. That said, I guess it's fine to tell the user that .position has axes not in .axes. I'll add some warnings.